src/lambda/handler-runner/go-runner/GoRunner.js

```diff
@@ -30,7 +30,7 @@ export default class GoRunner {
     }
 
     // Make sure we have the mock-lambda runner
-    sync('go', ['get', 'github.com/icarus-sullivan/mock-lambda@e065469'])
```

Comment:
For visibility, this is the commit hash for the mock-lambda changes https://github.com/icarus-sullivan/mock-lambda/commit/b33d704534c93aca178a559d934570e398ded646